src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailsChart.tsx

```diff
@@ -10,27 +10,13 @@ import { aggregateNodes } from '../../../../app/details/appDetails/utils'
 import { STATUS_SORTING_ORDER } from './constants'
 
 export default function AppStatusDetailsChart({
-    appStreamData,
+    resourcesSyncResult,
     filterRemoveHealth = false,
     showFooter,
 }: AppStatusDetailsChartType) {
     const _appDetails = IndexStore.getAppDetails()
-    const [nodeStatusMap, setNodeStatusMap] = useState<Map<string, NodeStreamMap>>()
     const [currentFilter, setCurrentFilter] = useState('')
 
-    useEffect(() => {
```

Comment:
Is this removal tested out

Reply:
Yes

Reply:
please resolve all sonarcloud issues & merge with main

Reply:
Sonar issues resolved